ethcore/src/types/basic_account.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Basic account type -- the decoded RLP from the state trie.
+
+use rlp::*;
+use util::{U256, H256};
+
+/// Basic account type.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct BasicAccount {
+	/// Nonce of the account.
+	pub nonce: U256,
```

Comment:
Ah, I think it's the only way to create `ACC_EMPTY`, right? Still I don't really like that you can modify internals freely.